docs/src/main/docs/guides/91_mp-tutorial.adoc

```diff
@@ -0,0 +1,1267 @@
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Helidon MP Tutorial
+:description: Helidon MP Tutorial
+:keywords: helidon
+
+This tutorial describes how to build an Helidon MP application from scratch 
+ including JSON REST endpoints, metrics, health check, and configuration.
+
+== What you need
+
+[width=50%,role="flex, sm7"]
+|===
+|About 30 minutes
+|<<about/03_prerequisites.adoc,Helidon Prerequisites>>
+|===
+
+== Create The Project
+
+This tutorial demonstrates how to create the application from scratch, without
+ using the Maven archetypes as a quickstart. 
+ 
+Create a new empty directory for the project (for example, `helidon-mp-tutorial`).  Change into this directory.
+
+Create a new Maven POM file (called `pom.xml`) and add the following
+ content: 
+ 
+[source,xml,subs="attributes+"]
+.Initial Maven POM file
+----
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.helidon.examples</groupId>
+    <artifactId>helidon-mp-tutorial</artifactId>  <1>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>${project.artifactId}</name>
+
+    <properties>
+        <helidon.version>{helidon-version}</helidon.version>
+        <mainClass>io.helidon.examples.Main</mainClass>
+        <maven.compiler.source>8</maven.compiler.source> <2>
+        <maven.compiler.target>${maven.compiler.source}</maven.compiler.target>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.helidon</groupId>
+                <artifactId>helidon-bom</artifactId> <3>
+                <version>${helidon.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.helidon.microprofile.bundles</groupId>
+            <artifactId>helidon-microprofile-2.2</artifactId> <4>
+        </dependency>
+    </dependencies>
+
+</project>
+----
+
+This POM files contains just the basics that we need to get started.  
+
+<1> It establishes the Maven coordinates for the new project.
+<2> The `properties` section tells Maven to use Java 8 for the source
+ and target versions, and sets the `mainClass` which will be used later
+ when building a JAR file.  The class will be created later in the
+ tutorial.
+<3> It includes the Helidon BOM POM  (in the `dependencyManagement` section).
```

Comment:
My experience is that 99 out of 100 users have no idea what a BOM POM is.  In general we have this issue: how much Maven basics do we explain?  I don't have a good answer.